src/pip/_internal/commands/list.py

```diff
@@ -329,7 +329,7 @@ def format_for_columns(
     for proj in pkgs:
         # if we're working on the 'outdated' list, separate out the
         # latest_version and type
-        row = [proj.raw_name, str(proj.version)]
+        row = [proj.raw_name, str(proj.version_str)]
```

Comment:
```suggestion
        row = [proj.raw_name, proj.version_str]
```

Since this is already a str? (Not entirely sure)